lib/rubocop/cop/root_cops/factory_file_name.rb

```diff
@@ -1,10 +1,10 @@
-require "pry"
+require "active_support/inflector"
 
 module RuboCop
   module Cop
     module RootCops
       class FactoryFileName < Cop
-        FILE_NAME_ERROR = "Factory name should match file name".freeze
+        FILE_NAME_ERROR = "Factory name should match singular form of file name".freeze
```

Comment:
I'm not too sure what the message should be for when the cop detects an error, because there are two situations that will lead to an error:
1. *Problem:* There is one factory in a file, but its name does not match the file name
*Solution:* Change the name of the factory to match the file name

2.  *Problem:* There is more than one factory in a file, and they do not follow [this naming convention](https://github.com/Root-App/root-ruby-style/pull/9#discussion_r272742021) 
*Solution:* Either split up the factories so they each have their own file, or make the factories follow the `__` convention as mentioned above.